### ISSUE TITLE
Baseline model outperforms P7-model on a waste dataset with 4000 images

### ISSUE THREAD

**Issue by Padlu** [WongKinYiu/ScaledYOLOv4#317]
I finetuned the baseline and p7 versions of Scaled-yolov4 model on my custom waste dataset which only had 4000 images to check which performs better by what mAP value on my dataset. An obvious expectation was that P7 model will outperform the baseline version. But, shockingly, it turns out that baseline version performed way better than P7 model in terms of mAP, accuracy of classification, train time, inference time where both were trained over 100 epochs. I'm not able to understand the reason why is this the case in my experiment. I also performed the same style of experiment using EfficientDet models, and again baseline EfficientDet-d0 outperformed EfficientDet-d7x. 

Does anyone has an explanation to this unexpected results in my experiment? Thanks. @WongKinYiu
